base/sysimg.jl

```diff
@@ -180,6 +180,27 @@ big(n::Integer) = convert(BigInt,n)
 big(x::FloatingPoint) = convert(BigFloat,x)
 big(q::Rational) = big(num(q))//big(den(q))
 
+const _fact_table128 =
```

Comment:
I don't think you should put these lines here. This file contains almost exclusively includes. Have you tried simply moving `include("combinatorics.jl")` below the BigInt section?